src/components/SSidebarItem.vue

```diff
@@ -0,0 +1,61 @@
+<script>
+export default {
+  name: 'SSidebarItem'
+}
+</script>
+
+<script setup>
+const props = defineProps({
+  text: {
+    type: String,
+    required: true
+  },
+  icon: {
+    type: Object,
+    required: true
+  }
+})
+</script>
+
+<template>
+  <div class="sidebar-item">
+    <a href="/">
+      <h1 class="text"> {{ text }}
+       <component class="=link-icon" :is="icon" />
+      </h1> 
+    </a>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+a:visited {
+  color: #91969e;
+}
+
+a:hover {
+  color: #c41f32;
+}
+
+a:link {
+  text-decoration: none;
+}
```

Comment:
You can use sass magic here by writing it like this:

```suggestion
a {
  &:visited {
    color: #91969e;
  }

  &:hover {
    color: #c41f32;
  }

  &:link {
    text-decoration: none;
  }
}
```